package.json

```diff
@@ -4,20 +4,27 @@
   "private": true,
   "heroku config": "set NPM_CONFIG_PRODUCTION=false",
   "dependencies": {
+    "@ckeditor/ckeditor5-build-classic": "^11.1.1",
+    "@ckeditor/ckeditor5-build-decoupled-document": "^11.1.1",
+    "@ckeditor/ckeditor5-build-inline": "^11.1.1",
+    "@ckeditor/ckeditor5-react": "^1.0.0",
     "axios": "^0.18.0",
     "classnames": "^2.2.6",
     "dotenv-webpack": "^1.5.7",
     "file-loader": "^2.0.0",
+    "font-awesome": "^4.7.0",
     "history": "latest",
     "jimp": "^0.5.4",
     "jquery": "^3.3.1",
     "jwt-decode": "^2.2.0",
+    "moment": "^2.22.2",
     "prop-types": "latest",
     "react": "^16.5.2",
+    "react-chips": "^0.8.0",
```

Comment:
I am not sure, but if this is used for tags, kindly remove it

Reply:
Ok @isheebo